op-program/client/l2/engineapi/precompiles.go

```diff
@@ -66,6 +73,75 @@ func CreatePrecompileOverrides(precompileOracle PrecompileOracle) vm.PrecompileO
 			return &precompile
 		case kzgPointEvaluationPrecompileAddress:
 			return &kzgPointEvaluationOracle{Orig: orig, Oracle: precompileOracle}
+		case blsG1AddPrecompileAddress:
+			// no size limit - fixed input
+			return &blsOperationOracle{
+				Orig:              orig,
+				Oracle:            precompileOracle,
+				checkInputSize:    checkInputExactSize(256),
+				checkOutput:       checkOutputExactSize(128),
+				precompileAddress: blsG1AddPrecompileAddress,
+			}
+		case blsG1MSMPrecompileAddress:
+			return &blsOperationOracleWithSizeLimit{
+				sizeLimit: 513760,
```

Comment:
Let's add a TODO to replace the size limit with the protocol constants in op-geth.